python/paddle/fluid/distribute_transpiler.py

```diff
@@ -500,38 +484,83 @@ def _clone_var(self, block, var):
             lod_level=var.lod_level,
             persistable=True)
 
-    def _append_split_op(self, program, gradblocks):
+    def _find_op_by_out(self, program, var):
+        for idx, op in enumerate(program.global_block().ops):
+            if var.name in op.output_arg_names:
+                return idx + 1
+        return -1
+
+    def _insert_send_vars_op(self, program, index, send_vars, epmap, eps):
```

Comment:
This function seems not used, are we planning to use it?

Reply:
Delete the unused function, done.